.env

```
@@ -352,7 +352,8 @@ VITE_YIELD_XYZ_API_KEY=06903960-e442-4870-81eb-03ff3ad4c035
 
 # Across Protocol
 VITE_ACROSS_API_URL=https://app.across.to/api
-VITE_ACROSS_INTEGRATOR_ID=
+VITE_ACROSS_INTEGRATOR_ID=0x021c
+VITE_ACROSS_API_KEY=acx_UsQJEgNekMMkqXP9KMD3Ol7hqRrP8Yrf
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ACROSS_API_KEY key should go before the VITE_ACROSS_API_URL key

(UnorderedKey)